language/stdlib/modules/libra_account.mvir

```diff
@@ -14,6 +14,9 @@ module LibraAccount {
         authentication_key: bytearray,
         // The coins stored in this account
         balance: LibraCoin.T,
+        // If true, the authority to rotate the authentication key of this account resides elsewhere
```

Comment:
Would it be possible to preserve the ability to rotate auth key also for the account owner? That way we could still do the periodic proactive self key rotation using the hot key itself but also have a backup mechanism to rotate in the delegated account in case the hot key is compromised and rotated by malicious actors.

Reply:
I agree, both the auth_key and cold_rotation_key should be able to rotate the `auth_key`.

I guess when used in practice some implementations would allow optionality on selecting the desired policy, but let's better keep it simple for this example.

Also, if one uses both capabilities, we should mention somewhere that controlling the rotation capability does not necessarily give you spending power. Good implementations should be carefully designed to  avoid such edge-cases, i.e., by including both capabilities in the module logic.

Can we also add a comment or test on what happens if there is a rotation cycle A->B->A or A->B->C->A?

Reply:
The use-case @weilei0107 has in mind can indeed be supported with the existing code (and I have changed the E2E test to demonstrate this).

Good point about the odd cases that can come up with using both the withdrawal and rotation capabilities + cycles. I'd like to spend some more time thinking about/documenting the best practices + writing tests after getting this change in. I think this will fit nicely with trying to write formal specifications for our hot/cold wallets (@DavidLDill will be excited).